oam/install_scripts/columnstore-pre-uninstall.in

```diff
@@ -49,8 +49,8 @@ if [ -n "$(pgrep -x ProcMon)" ] || [ -n "$(pgrep -x ProcMgr)" ];then
 fi
 
 rm -f /etc/profile.d/columnstoreAlias.sh
-rm -f /var/log/mariadb/columnstore/activeAlarms > /dev/null 2>&1
-rm -f /var/log/mariadb/columnstore/*.log1 > /dev/null 2>&1
+rm -f @ENGINE_LOGDIR@/activeAlarms > /dev/null 2>&1
```

Comment:
Not sure this makes sense b/c there was no ActiveAlarms neighter in 1.5 nor in 5.x . There might be either 1.4 or 1.2 that has it and 1.4 has in /var/log/mariadb/columnstore only. 1.2 has it in /usr/local...

Reply:
Agreed, will remove but this should be harmless as its only making sure it was removed on uninstall.